tee-worker/ts-tests/worker/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "stress",
+    "license": "ISC",
+    "type": "module",
+    "scripts": {
+        "format": "pnpm exec prettier --write .",
+        "check-format": "pnpm exec prettier --check .",
+        "run-script": "NODE_TLS_REJECT_UNAUTHORIZED=0 pnpm exec ts-node bin/stress.ts",
```

Comment:
Are these still scripts related to stress testing? Shouldn't be related to the worker?

Reply:
Hmm it looks like my local changes weren't included in this commit. I'll take a look